packages/nest-pg-notify/src/lib/pg-notify.constants.ts

```diff
@@ -0,0 +1,11 @@
+export const PG_NOTIFY_CHANNEL = 'pg-notify:channel' as const;
+export const PG_NOTIFY_NOTIFICATION = 'pg-notify:notification' as const;
+
+export const PG_NOTIFY_CONFIG = 'pg-notify:config' as const;
+export const PG_NOTIFY_ASYNC_CONFIG = 'pg-notify:async-config' as const;
+export const PG_NOTIFY_SUBSCRIBER = 'pg-notify:subscriber' as const;
+
+export const PG_NOTIFY_REDIS = 'pg-notify:redis' as const;
```

Comment:
It might be better to use Symbols for injection tokens as there is no possibility of conflicts like there is with strings.